src/main/java/com/example/demo/TableController.java

```diff
@@ -0,0 +1,20 @@
+package com.example.demo;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.demo.Service.TablesService;
+
+@RestController
+public class TableController {	
```

Comment:
@yuyamatsymura 
質問ですがコントローラ名は単数形でよかった？
```suggestion
public class TablesController {	
```